src/main/kotlin/uk/gov/justice/digital/hmpps/riskprofiler/services/SocDecisionTreeService.kt

```diff
@@ -89,7 +89,7 @@ class SocDecisionTreeService(
 
   private fun isHasActiveSocAlerts(nomsId: String?): Boolean {
     return nomisService.getSocListAlertsForOffender(nomsId).stream()
-      .anyMatch { alert -> alert.active!! && alert.dateCreated!!.isAfter(LocalDate.now().minusYears(1)) }
+      .anyMatch { alert -> alert.active!! && !alert.expired!! && alert.dateCreated!!.isAfter(LocalDate.now().minusYears(1)) }
   }
```

Comment:
could expired by defined as not nullable if it always has a value, rather than !!  ?